Dockerfile.template

```diff
@@ -28,13 +28,17 @@ RUN apk add --update -t build-deps build-base git zlib-dev openssl-dev \
  && rm -rf /usr/src/*
 
 # Mirror Websites
-ENV SITES https://5pi.de
+ENV SITES http://elpissite.weebly.com
 RUN mkdir -p /content/www \
  && cd /content/www \
- && httrack $SITES \
+ && httrack --ext-depth=1 $SITES \
  && find . -type f -maxdepth 1 -delete \
  && rm -rf hts-cache
 
+#Add CSS to html files to hide the footer
+RUN cd /content/www/elpissite.weebly.com \
```

Comment:
Can you merge this into the RUN statement above?
Each RUN statement creates a new image layer. If you change something, it will not actually change the files but store the differ in this layer which increases the file size unnecessary.